main.py

```diff
@@ -0,0 +1,12 @@
+from src.proccess_csv import process_data
+from sklearn.metrics.pairwise import cosine_similarity
+
+if __name__ == "__main__":
+    df_movies, columndictionary = process_data()
+    genre_columns = df_movies.columns[24:]
+
+    user_preference = [1, 0, 0, 0, 0, 0, 1, 1, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0]
```

Comment:
What is the user preference in this case?

Reply:
The preferred genres, this is only a test for now, I'll add later to prompt the user the preferred genres and mask them to the array